README.md

```diff
@@ -1,77 +1,134 @@
-# ODO - OpenShift Developer Optimization
+# Odo - OpenShift do
 
-[![Build Status](https://travis-ci.org/redhat-developer/odo.svg?branch=master)](https://travis-ci.org/redhat-developer/odo) [![codecov](https://codecov.io/gh/redhat-developer/odo/branch/master/graph/badge.svg)](https://codecov.io/gh/redhat-developer/odo)
+[![Build Status](https://travis-ci.org/redhat-developer/odo.svg?branch=master)](https://travis-ci.org/redhat-developer/odo) [![codecov](https://codecov.io/gh/redhat-developer/odo/branch/master/graph/badge.svg)](https://codecov.io/gh/redhat-developer/odo) [![GoDoc Widget]][GoDoc]
```

Comment:
I'm not sure if it is a good idea to add a link to Godoc reference. The output of godoc is pretty useless right now :-(